.mend/notes/xbrlkit-scout-report.md

```
@@ -0,0 +1,276 @@
+| SCN-XK-SEC-DECIMAL-010 | sec/decimal_precision.feature | AC-XK-SEC-DECIMAL-002 |
+| SCN-XK-SEC-INLINE-001 | sec/inline_restrictions.feature | AC-XK-SEC-INLINE-001 |
+| SCN-XK-SEC-INLINE-002 | sec/inline_restrictions.feature | AC-XK-SEC-INLINE-002 |
+| SCN-XK-SEC-NEGATIVE-001 | sec/negative_values.feature | AC-XK-SEC-NEGATIVE-001 |
+| SCN-XK-SEC-NEGATIVE-002 | sec/negative_values.feature | AC-XK-SEC-NEGATIVE-002 |
+| SCN-XK-SEC-NEGATIVE-003 | sec/negative_values.feature | AC-XK-SEC-NEGATIVE-003 |
+| SCN-XK-SEC-NEGATIVE-004 | sec/negative_values.feature | AC-XK-SEC-NEGATIVE-004 |
+| SCN-XK-SEC-NEGATIVE-005 | sec/negative_values.feature | AC-XK-SEC-NEGATIVE-005 |
+| SCN-XK-SEC-REQUIRED-001 | sec/required_facts.feature | AC-XK-SEC-REQUIRED-001 |
+| SCN-XK-SEC-REQUIRED-002 | sec/required_facts.feature | AC-XK-SEC-REQUIRED-002 |
+
+### Workflow (5 scenarios)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)